src/cls/rbd/cls_rbd_client.h

```diff
@@ -285,11 +285,14 @@ namespace librbd {
     int mirror_uuid_set(librados::IoCtx *ioctx, const std::string &uuid);
     void mirror_mode_get_start(librados::ObjectReadOperation *op);
     int mirror_mode_get_finish(bufferlist::iterator *it,
-			       cls::rbd::MirrorMode *mirror_mode);
+			       cls::rbd::MirrorMode *mirror_mode,
+			       std::string *data_pool_name = nullptr);
```

Comment:
I added default values here to limit the size of the change, but I'm not sure if there's precedence for this in the cls code.